### ISSUE TITLE
Android: can't install signed package

### ISSUE THREAD

**Issue by retyui** [microsoft/react-native-code-push#2426]
Thanks so much for filing an issue or feature request! Please fill out the following (wherever relevant):

### Steps to Reproduce
1. 
2. add code signing / or git clone [demo](https://github.com/retyui/CodePushCiCd/commit/879f2494fa1ae1bce79fdc23d6dc88625b25e08a)
3. run android
4. open dev menu
5. tap on `certs` package 

### Expected Behavior

works well

### Actual Behavior

see:

https://user-images.githubusercontent.com/4661784/213273543-e0535a83-162d-474e-9c5b-87c298f6e49e.mp4

```
[CodePush] Exception
com.microsoft.codepush.react.CodePushUnknownException: Error in getting binary resources modified time
	at com.microsoft.codepush.react.CodePush.getBinaryResourcesModifiedTime(CodePush.java:207)
	at com.microsoft.codepush.react.CodePushNativeModule$3.doInBackground(CodePushNativeModule.java:289)
	at com.microsoft.codepush.react.CodePushNativeModule$3.doInBackground(CodePushNativeModule.java:284)
	at android.os.AsyncTask$3.call(AsyncTask.java:394)
	at java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1137)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:637)
	at java.lang.Thread.run(Thread.java:1012)
Caused by: android.content.res.Resources$NotFoundException: String resource ID #0x0
	at android.content.res.Resources.getText(Resources.java:453)
	at android.content.res.Resources.getString(Resources.java:546)
	at com.microsoft.codepush.react.CodePush.getBinaryResourcesModifiedTime(CodePush.java:204)
	... 7 more

```

### Reproducible Demo

- https://github.com/retyui/CodePushCiCd/commit/879f2494fa1ae1bce79fdc23d6dc88625b25e08a

### Environment

* react-native-code-push version: 7.0.5
* react-native version: 0.70.6
* iOS/Android/Windows version: Android
* Does this reproduce on a debug build or release build? both
* Does this reproduce on a simulator, or only on a physical device? both

- https://github.com/microsoft/react-native-code-push/blob/master/docs/setup-android.md#code-signing-setup
- https://learn.microsoft.com/en-us/appcenter/distribution/codepush/cli#code-signing

**Comment by ko-devHong** [comment]
it fix issue : version `7.1.0`

ref : 
 https://github.com/microsoft/react-native-code-push/pull/2337 
https://github.com/microsoft/react-native-code-push/issues/2116

**Comment by retyui** [comment]
Update to the latest ver. fixed the issue, thanks!
